### PR TITLE
fix: team channel reload error message

### PR DESCRIPTION
**Pull request by yash-rajpal**
## Steps to test or reproduce

1. create a team - A
2. create a channel - B.
3. Add existing channel B to team A.
4. Red toast bar appears
## Further comments

[TC-432]

[TC-432]: https://rocketchat.atlassian.net/browse/TC-432?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dougfabris — CHANGES_REQUESTED** [review]
Dude, I would like to ask you to hold this PR until I got this one merged: #28666

**Review by dougfabris — CHANGES_REQUESTED** [review]
@yash-rajpal let's avoid doing some assertions using the `ToastBar` just verify if the room is on the list, is enough!

**Review by dougfabris — DISMISSED** [review]

**Review by juliajforesti — APPROVED** [review]

**Review by tassoevan — APPROVED** [review]

**Pull request merged**
